msg/templates/uorb_microcdr/microRTPS_client.cpp.template

```diff
@@ -120,7 +112,7 @@ void* send(void* /*unused*/)
                 /* payload is shifted by header length to make room for header*/
                 serialize_@(topic)(&MicroBufferWriter, &data, &data_buffer[header_length], &length);
 
-                if (0 < (read = transport_node->write((char)@(rtps_message_id(msg_ids, topic)), data_buffer, length)))
+                if (0 < (read = transport_node->write((char)@(rtps_message_id(ids, topic)), data_buffer, length)))
```

Comment:
Can you add `ids` to the Context documentation (https://github.com/PX4/Firmware/pull/10548/files#diff-6bb9665760d1bd254be1bfe644bc94c2R8)? (for all the templates)

Reply:
done